README.md

```diff
@@ -82,6 +82,7 @@ const ele = syntaxHighlighter('console.log("Hello, world!");', 'js', { ...opts,
 | Java | `java` |
 | JavaScript | `coffeescript`, `ecmascript`, `javascript`, `js`, `node` |
 | JSON | `json` |
+| JSX | `jsx` |
```

Comment:
We probably need to update the RDMD docs too!

Reply:
Will need to bump RDMD as well to pull in support for highlighting JSX codeblocks there too.